coordinator/src/main/scala/filodb.coordinator/queryplanner/PlannerMaterializer.scala

```diff
@@ -141,23 +139,3 @@ trait  PlannerMaterializer {
       }
     }
 }
-
-object PlannerUtil extends StrictLogging {
-
-  val bjBetweenAggAndNonAgg = Kamon.counter("join-between-agg-non-agg").withoutTags()
-
-  def validateBinaryJoin(lhs: Seq[ExecPlan], rhs: Seq[ExecPlan], queryContext: QueryContext): Any = {
-
-    if (lhs.exists(_.isInstanceOf[LocalPartitionReduceAggregateExec]) &&
-      !rhs.exists(_.isInstanceOf[LocalPartitionReduceAggregateExec])) {
-      logger.info(s"Saw Binary Join between aggregate(lhs) and non-aggregate (rhs). ${queryContext.origQueryParams}")
-      bjBetweenAggAndNonAgg.increment()
-    }
-
-    if (!lhs.exists(_.isInstanceOf[LocalPartitionReduceAggregateExec]) &&
-      rhs.exists(_.isInstanceOf[LocalPartitionReduceAggregateExec])) {
-      logger.info(s"Saw Binary Join between non-aggregate(lhs) and aggregate(rhs): ${queryContext.origQueryParams}")
-      bjBetweenAggAndNonAgg.increment()
-    }
-  }
-}
```

Comment:
These changes are not in prod right , would it make sense to create a new pr , with a more clean change log so that it is easier to cherry pick